### DISCUSSION TITLE
target="_blank" só serve para abrir páginas em uma nova aba?

### DISCUSSION THREAD

**Discussion by fdaciuk** [frontendbr/forum#87] [Dicas]
Eu também achava isso, mas encontrei hoje [esse artigo](http://mathiasbynens.github.io/rel-noopener/) que explica os **perigos** de usar só o `target="_blank"`.

Alguém já conhecia o `rel=noopener`?

**Comment by lucasbento** [comment]
Caramba, não conhecia e não tinha ideia disso.

**Comment by vitor-mendrone-deel** [comment]
E a poucos dias eu abri um tread por conta do target="_blank" aqui no fórum mas por questões de usabilidade e acessibilidade. 

Essa é nova, obrigado por compartilhar @fdaciuk!

**Comment by williamcosta** [comment]
Muito bom o artigo. Também não conhecia!

**Comment by felipefialho** [comment]
Bem interessante, não fazia ideia também.

**Comment by RatoX** [comment]
Boa, não conhecia, obrigado por compartilhar

**Comment by danilovaz** [comment]
@fdaciuk  Cara, que coisa! :confused: Desconhecia isso, também. Algo "besta" que pode impactar a aplicação.

**Comment by ramonsenadev** [comment]
Valeu!!! Também não sabia disso. Mais uma vez os detalhes.. hah

**Comment by fabiomirandaa** [comment]
@raphaklaus Olha essa.. Simples e boa essa questão

**Comment by renatogalvones** [comment]
Sensacional @fdaciuk !!!

Valeu por sharear!

**Comment by JoseCage** [comment]
Super essa dica @fdaciuk 

Valeu.. :+1:

**Comment by ViniciusRod** [comment]
Dica ótima!

**Comment by tarciozemel** [comment]
Conhecia, sim.

Há algum tempo, coloquei uma tradução sobre isso no blog em que escrevo: https://desenvolvimentoparaweb.com/miscelanea/relnoopener-performance-seguranca/

**Comment by CavalcanteLeo** [comment]
Firefox não suporta a tag noopener, então você deve usar a seguinte tag: rel=”noopener noreferrer”

**Comment by Soliun** [comment]
> 
> 
> Firefox não suporta a tag noopener, então você deve usar a seguinte tag: rel=”noopener noreferrer”

Firefox 79 vai ter suporte: https://www.fxsitecompat.dev/en-CA/docs/2020/target-blank-on-anchors-now-implies-rel-noopener/
